test/integration/main_test.go

```diff
@@ -17,11 +17,32 @@ limitations under the License.
 package integration
 
 import (
+	"log"
+	"path/filepath"
 	"testing"
 
-	"k8s.io/kubernetes/test/integration/framework"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
 )
 
+var testCfg *rest.Config
```

Comment:
Add a comment above, and maybe rename to globalKubeConfig

```
// Global kubeConfig to connect to the testing API server.
```

Reply:
Sure thing, done.